integration-tests/src/test/resources/wdt-models/model-fmwdomain-onpv-wdt.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+domainInfo:
+    AdminUserName: '@@PROP:adminUsername@@'
+    AdminPassword: '@@PROP:adminPassword@@'
+    ServerStartMode: 'prod'
+    RCUDbInfo:
+        rcu_prefix: '@@PROP:rcuSchemaPrefix@@'
+        rcu_schema_password: '@@PROP:rcuSchemaPassword@@'
+        rcu_db_conn_string: '@@PROP:rcuDb@@'
+
+topology:
+    AdminServerName: "admin-server"
+    Name: "fmw-domaininimage-inpv"
+    Log:
+        FileName: domain1.log
+    NMProperties:
+        JavaHome: /usr/java/jdk1.8.0_211
```

Comment:
Why all hard-coded location ?  I have not noticed such configuration in any model yaml file.  NodeManager is internal to Operator framework. It should not be exposed to customer thru model file.

Reply:
fixed